sample-functions/BaseFunctions/R/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM artemklevtsov/r-alpine:latest
+
+ADD https://github.com/alexellis/faas/releases/download/0.5.1-alpha/fwatchdog /usr/bin
```

Comment:
It would be great if you could change `0.5.1-alpha` to `0.6.2` (without `alpha`).

Reply:
Ok. I changed